### PR TITLE
Remove duplicate quote and fix quote `id` increment

### PR DESCRIPTION
**Pull request by UnrealApex**
Resolves [#4126](https://github.com/monkeytypegame/monkeytype/discussions/4126)

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -36876,1924 +36876,1917 @@
...
-      "id": 6527
+      "id": 6526
     },
     {
       "text": "Well, the way I see it, one of two things can happen. Either we'll discover that we've foolishly built each other up in our own imaginations because we've had so much trouble getting together; or, this could end up being a story we tell our grandchildren. Personally, I can't wait to find out which.",
       "source": "Frasier",
       "length": 299,
       "approvedBy": "Smithster",
-      "id": 6528
+      "id": 6527
```
Comment (UnrealApex) [inline frontend/static/quotes/english.json:36947]: Looks like someone before me screwed up the ids. I will fix this

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -36876,1924 +36876,1917 @@
...
     },
     {
       "text": "There was me, that is Alex, and my three droogs, that is Pete, Georgie, and Dim, and we sat in the Korova Milkbar trying to make up our rassoodocks what to do with the evening. The Korova milkbar sold milk-plus, milk plus vellocet or synthemesc or drencrom, which is what we were drinking. This would sharpen you up and make you ready for a bit of the old ultra-violence.",
       "source": "Alex DeLarge, A Clockwork Orange ",
       "length": 371,
       "approvedBy": "Smithster",
-      "id": 6530
+      "id": 6529
```
Comment (UnrealApex) [inline frontend/static/quotes/english.json:36954]: Looks like someone before me screwed up the ids. I will fix this

**Review by Ferotiq — APPROVED** [review]

**Comment by Miodec** [comment]
Reverting. @Ferotiq @UnrealApex dont change quote ID's - it will mess up favorites and ratings.

**Comment by UnrealApex** [comment]
Perhaps I should have consulted you first about this change.  Taking what you said into consideration, how should duplicates be handled? Should they just be left as they are?

**Comment by Miodec** [comment]
Yes

**Comment by UnrealApex** [comment]
Okay, thanks for the clarification! I will create a new PR

**Pull request merged**